Formula/m/mesa.rb

```diff
@@ -154,6 +153,9 @@ def install
         -Dvulkan-drivers=amd,intel,intel_hasvk,swrast,virtio
         -Dvulkan-layers=device-select,intel-nullhw,overlay
       ]
+      if Hardware::CPU.intel?
+        args << "-Dgallium-drivers=r300,r600,radeonsi,nouveau,virgl,svga,swrast,i915,iris,crocus,zink"
+      end
```

Comment:
```suggestion
      if Hardware::CPU.intel?
        args << "-Dgallium-drivers=r300,r600,radeonsi,nouveau,virgl,svga,swrast,i915,iris,crocus,zink"
      else 
        args << "-Dgallium-drivers=virgl,svga,swrast,zink"
      end
```

Reply:
And `zink` is also a software driver that implements OpenGL on top of Vulkan, which may be supplied by a vendor, not by mesa.